### PR TITLE
rart67m: move OLED and WPM code to default keymap

### PR DESCRIPTION
**Pull request by fauxpark**
## Description

Can't be built in Configurator due to `WPM_ENABLE` only being enabled at keymap level.
image

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/rart/rart67m/keymaps/default/keymap.c [hunk 1]
```diff
@@ -14,36 +14,132 @@
...
+        oled_write_raw_P(prep[0], ANIM_SIZE);
+    }
+
+    if (get_current_wpm() >= TAP_SPEED) {
+        current_tap_frame = (current_tap_frame + 1) % TAP_FRAMES;
+        oled_write_raw_P(tap[abs((TAP_FRAMES - 1) - current_tap_frame)], ANIM_SIZE);
+    }
+}
+
+oled_rotation_t oled_init_kb(oled_rotation_t rotation) {
```
Comment (zvecr) [inline keyboards/rart/rart67m/keymaps/default/keymap.c:113]:
```suggestion
oled_rotation_t oled_init_user(oled_rotation_t rotation) {
```

File: keyboards/rart/rart67m/keymaps/default/keymap.c [hunk 2]
```diff
@@ -14,36 +14,132 @@
...
+    return OLED_ROTATION_180;
+}
+
+bool oled_task_kb(void) {
+    if (!oled_task_user()) {
+        return false;
+    }
+
```
Comment (zvecr) [inline keyboards/rart/rart67m/keymaps/default/keymap.c:121]:
```suggestion
bool oled_task_user(void) {
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Comment by drashna** [comment]
why not enable wpm by default, or add preprocessor around the wpm dependant code?

**Comment by fauxpark** [comment]
I'd rather leave WPM off by default to save space, and the preprocessor stuff was just too messy.

**Review by zvecr — APPROVED** [review]

**Pull request merged**